packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -96,6 +96,39 @@ const toCSSDeclaration = (key: string, result: CSSOutput) => {
   return result.css.map((x) => ({ ...x, css: `${kebabCase(key)}: ${getItemCss(x)};` }));
 };
 
+/**
+ * Will return the variable declarator value node from meta's own path.
+ * It will compare the variable declarator node id's name with passed node name and
+ * returns its stringified value along with the node.
+ *
+ * Eg. If passed node is an Identifier with name `abcd` and somewhere in own path
+ * we have any variable declarator `const abcd = obj.value`, it will return
+ * stringified `obj.value` along with node `obj.value`.
+ * If variable declarator's value is undefined `const abcd = undefined`, it will
+ * return stringified `abcd` along with undefined.
+ *
+ * @param node
+ * @param meta Meta data used during the transformation.
+ */
+const getVariableDeclaratorValueForOwnPath = (node: t.Expression, meta: Metadata) => {
+  let variableName = generate(node).code;
```

Comment:
can you add a comment for what this will produce? for example does this make the stringified "obj.value" i guess?

(as a nit: any way we can avoid needing to generate the code?)